README.md

```diff
@@ -54,6 +54,32 @@ genhtml -o report/ --show-details --highlight --ignore-errors source --legend lc
 grcov ~/Documents/FD/mozilla-central/build -t coveralls -s ~/Documents/FD/mozilla-central --token YOUR_COVERALLS_TOKEN > coveralls.json
 ```
 
+### GRCOV with Travis
+
+Here is an example of .travis.yml file
+```YAML
+sudo: false
+language: rust
+
+before_install:
+  - curl -L https://github.com/mozilla/grcov/releases/download/v0.4.1/grcov-linux-x86_64.tar.bz2 | tar jxf -
+
+matrix:
+  include:
+    - os: linux
+      rust: nightly
+
+script:
+    - export CARGO_INCREMENTAL=0
+    - export RUSTFLAGS="-Zprofile -Ccodegen-units=1"
+    - cargo build --verbose $CARGO_OPTIONS
+    - cargo test --verbose $CARGO_OPTIONS
+    - |
+      zip -0 ccov.zip `find . \( -name "YOUR_PROJECT_NAME*.gc*" \) -print`;
```

Comment:
What if we remove `YOUR_PROJECT_NAME*` and just find all files ending with gcda or gcno?

Or maybe we can just execute grcov on `.`, without zipping anything.

Reply:
there are a lot of gcno, gcda for dependencies we don't have source files in the repo so it's useless to have them.